x-pack/test/api_integration/apis/slos/fixtures/create_slo.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "Test SLO for api integration",
+    "description": "Fixture for api integration tests",
+    "indicator": {
+      "type": "sli.kql.custom",
+      "params": {
+        "index": "kbn-data-forge*",
+        "filter": "system.network.name: eth1",
+        "good": "container.cpu.user.pct < 1",
+        "total": "container.cpu.user.pct: *",
+        "timestampField": "@timestamp"
+      }
+    },
+    "budgetingMethod": "occurrences",
+    "timeWindow": {
+      "duration": "7d",
+      "type": "rolling"
+    },
+    "objective": {
+      "target": 0.99
+    },
+    "tags": [
+      "test"
+    ],
+    "groupBy": "tags"
+}
```

Comment:
can you please just use a simple ts file? that way you can also use ts types and any changes can be quickly caught. i don't see a particulat advantage to using .json file.

Reply:
Sure thing!

Reply:
I forgot to change this. I can do so in a follow up.